api/src/test/java/org/pmiops/workbench/reporting/insertion/InsertAllRequestBuilderTest.java

```diff
@@ -130,6 +129,5 @@ public void testWorkspace_allFields() {
         .hasSize(FIXED_VALUES.size() + WorkspaceColumnValueExtractor.values().length);
     assertThat(contentMap.get("cdr_version_id")).isEqualTo(WORKSPACE__CDR_VERSION_ID);
     assertThat(contentMap.get("name")).isEqualTo(WORKSPACE__NAME);
-    assertThat(contentMap.get("published")).isEqualTo(WORKSPACE__PUBLISHED);
```

Comment:
this only tests a few fields, so I don't think adding the featured category is necessary